test/data2/local_config.json

```diff
@@ -16,7 +17,11 @@
     "operator": "cosmos1ddvpnk98da5hgzz8lf5y82gnsrhvu3jd3cukpp",
     "passphrase": "12341234",
     "pubkey": "cosmosvalconspub1zcjduepqmnz2m5asxrqplfl52lm2meq8fq3hc562ljmc6lqawr6kzsmw2g3snvmrdy",
-    "transactors": ["cosmos104gnaehlj7a32r9l5za7nhf463yvgnyfnwh9sc"],
-    "timeoutCommit": 5
+    "transactors": [
+      "cosmos104gnaehlj7a32r9l5za7nhf463yvgnyfnwh9sc",
+      "cosmos1sev3nak38elq95lumnh6t2drjtfx73274vnnjh"
+    ],
+    "keyringBackend": "test",
+    "timeoutCommit": 2
   }
 }
```

Comment:
why do we need this folder? this file is a duplicate of data/local_config.json

Reply:
It is duplidated